.github/workflows/e2e-nextjs.yml

```
@@ -2,9 +2,9 @@ name: Next.js Integration test
 
 on:
   pull_request:
-    branches: [main]
-    # Run on release PRs only
-    paths: ['CHANGELOG.md']
+    branches: [main, 'chore/fix-ci-npm-cache']
```

Should we remove this change before merging?

---

We could maybe keep this but under a different name, what do you think? 👇 

```suggestion
    branches: [main, 'chore/ci-e2e-nextjs']
```